devtools/gqltsgen/run.go

```diff
@@ -65,16 +73,60 @@ func main() {
 
 	fmt.Fprintf(w, "// Code generated by devtools/gqltsgen DO NOT EDIT.\n\n")
 
-	for _, def := range doc.Definitions {
+	var types []*ast.Definition
+	for _, typ := range sch.Types {
```

Comment:
sch.Types is a map, so putting it into a slice so we can sort it, ensuring consistent output